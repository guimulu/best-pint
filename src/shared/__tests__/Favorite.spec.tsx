import * as React from "react";
import { createSnapshot } from "jest-react-fela";

import { Favorite } from "..";

describe("Favorite component", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Favorite />);
    expect(snapshot).toMatchSnapshot();
  });
});

describe("Favorite component red", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Favorite isFavorite />);
    expect(snapshot).toMatchSnapshot();
  });
});
