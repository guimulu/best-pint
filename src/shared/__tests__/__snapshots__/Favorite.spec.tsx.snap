// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorite component red renders correctly 1`] = `
".a {
  position: absolute;
}
.b {
  right: 16px;
}
.c {
  top: 0;
}
.d {
  font-size: 40px;
}
.e {
  color: #ed4f4f;
}
.h {
  transition: all 0.3s ease-in-out 0s;
}
.i::before {
  content: \\"♥\\";
}
.f:hover {
  cursor: pointer;
}
.g:hover {
  transform: scale(1.2);
}


<div className=a b c d e f g h i />;
"
`;

exports[`Favorite component renders correctly 1`] = `
".a {
  position: absolute;
}
.b {
  right: 16px;
}
.c {
  top: 0;
}
.d {
  font-size: 40px;
}
.e {
  color: #ededed;
}
.h {
  transition: all 0.3s ease-in-out 0s;
}
.i::before {
  content: \\"♥\\";
}
.f:hover {
  cursor: pointer;
}
.g:hover {
  transform: scale(1.2);
}


<div className=a b c d e f g h i />;
"
`;
